src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CommunityAPIService.kt

```diff
@@ -228,6 +233,7 @@ data class ReferRequest(
   val contractType: String,
   val startedAt: OffsetDateTime,
   val sentenceId: Long,
+  val referralId: UUID,
```

Comment:
Does it make sense to remove everything _except_ referral IDs?

Reply:
ReferralId is only used as a last resort if the existing criteria don't result in a unique match.

Reply:
But it can be used as known 1:1 mapping, right?

Reply:
Principally yes but I think searching via a notes field in the database is not a preferred approach. Currently, CRN Sentence and NSI Type (Contract Type) is the criteria for querying the database in community-api.

Reply:
I will merge this change for now and perhaps we can speak to @mxco86